samples/javascript-v4-azurefunction/src/functions/createNewOrder.js

```diff
@@ -0,0 +1,23 @@
+const { app, output, trigger } = require('@azure/functions');
+
+const daprStateOuput = output.generic({
+    type: "daprState",
+    stateStore: "%StateStoreName%",
+    direction: "out",
+    name: "order",
+    key: "order"
+});
+
+app.generic('CreateNewOrder', {
+    trigger: trigger.generic({
```

Comment:
Can't we add types here instead of using generic triggers and input? I am looking at examples from this blog post and I was hoping something similar for Dapr https://techcommunity.microsoft.com/t5/apps-on-azure-blog/azure-functions-version-4-of-the-node-js-programming-model-is-in/ba-p/3773541

Reply:
We can add those and considered it as well, but for now we were advised by node-js library owners to use generic until we see reasonable usage, or we get user feedback asking for it.

Reply:
Thanks for informing! We can wait for user feedback then.